src/gen_dragonfruit/gd_dev_apogee_sp215.cpp

```diff
@@ -21,7 +21,8 @@ int gd_dev_apogee_sp215_read(void){
         value += Wire.read();
     }
     Wire.endTransmission();
-    //value = (value*5000.00)/(0x7FFF);
+    /* Analog to digital conversion with 12 bit-resolution */   
+    value = (value*5000.00)/(0xFFF);
```

Comment:
What is the 0xFFF there for? Also, since the problem isn't operating at 12-bits, can you ramp back up to a 16-bit resolution again?